apps/meteor/app/livechat/server/methods/saveIntegration.ts

```diff
@@ -12,6 +13,7 @@ Meteor.methods({
 				method: 'livechat:saveIntegration',
 			});
 		}
+		methodDeprecationLogger.warn('livechat:saveIntegration will be deprecated in future versions of Rocket.Chat');
```

Comment:
```suggestion
		methodDeprecationLogger.warn('livechat:saveIntegration is deprecated and will be removed on the next major release of Rocket.Chat');
```

let's mark this as deprecated right away ;)

Reply:
deprecated